<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HiddenShots</class>
 <widget class="QWidget" name="HiddenShots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hidden Shots</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tab_hidden"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_unhide">
     <property name="text">
      <string>Unhide Slected Shots</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
